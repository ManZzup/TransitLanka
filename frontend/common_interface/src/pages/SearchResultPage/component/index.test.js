import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'
import { createStore,applyMiddleware } from 'redux'
import thunkMiddleware from 'redux-thunk'
import SearchResultPageComponent from './';
import rootReducer from '../../../reducers';

let store = createStore(rootReducer,applyMiddleware(
    thunkMiddleware
));

it('renders Search Results Page correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <SearchResultPageComponent startText="Pettah" endText="Moratuwa" />
    </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
