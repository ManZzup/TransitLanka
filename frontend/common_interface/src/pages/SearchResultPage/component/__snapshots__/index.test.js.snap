exports[`test renders Search Results Page correctly 1`] = `
<div
  className="">
  <div
    className="large-4 small-12 columns"
    style={
      Object {
        "boxShadow": "2px 1px 3px 1px #ccc",
        "height": "100vh",
      }
    }>
    <div
      style={Object {}}>
      <h1
        style={
          Object {
            "fontFamily": Array [
              "Open Sans Condensed",
              "sans-serif",
            ],
          }
        }>
         
        <font
          color="#1C6BA0">
          transit
        </font>
        <font
          color="#ffae00">
          Lanka
        </font>
      </h1>
      <div
        className="row">
        <div
          className="input-group">
          <span
            className="input-group-label">
            I am at
          </span>
          <input
            className="input-group-field awesomplete"
            defaultValue="Pettah"
            id="txt_start"
            onChange={[Function]}
            placeholder="Select start location"
            type="text" />
        </div>
      </div>
      <div
        className="row">
        <div
          className="input-group">
          <span
            className="input-group-label">
            I am going
          </span>
          <input
            className="input-group-field awesomplete"
            defaultValue="Moratuwa"
            id="txt_end"
            onChange={[Function]}
            placeholder="Select end location"
            type="text" />
        </div>
      </div>
    </div>
    <div
      className="row">
      <span>
        <input
          className="button warning"
          onClick={[Function]}
          type="button"
          value="Do MAGIC!" />
      </span>
      <span />
    </div>
  </div>
  <div
    className="large-7 small-12 columns end">
    <div
      className="small-12 columns"
      id="results"
      style={
        Object {
          "paddingTop": "10px",
        }
      }>
      <span />
    </div>
  </div>
</div>
`;
